Week2/homework/map_filter.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const numbers = [1, 2, 3, 4];
-
-// Add your code here
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+const newNumbers = numbers.filter(evenGoAway => 0 != evenGoAway % 2).map(multiplyIt => 2 * multiplyIt);
```

Comment:
The same comment on your week2 P1 homework. parameters naming is misleading, other than this good job 👍